db_scripts/db_dump

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+NAME=${1:-postgres}
+CONTAINER=$(docker ps --format "{{.Names}}" | grep "$NAME")
+if [ -z "$CONTAINER" ]
+	then
+		echo "Container not found"
+		exit 1
+fi
```

Comment:
We should probably also check that "$CONTAINER" contains exactly one result.
An easy way to do this would be to check that `[ "$(wc -l <<<"$CONTAINER")" -eq 1 ]`, ie. check there's exactly one line.